docs-user/guide-remote-profiling.md

```diff
@@ -119,11 +119,11 @@ If you want to profile an Android build that the tryserver created for you, you
 
 If you've compiled an Android Gecko build locally, and want to profile it, you have to jump through a few extra hoops at the moment unfortunately.
 
-At the moment, Firefox Nightly Desktop doesn't support symbolicating profiles that were obtained from local Android Gecko builds. However, [patches are](https://bugzilla.mozilla.org/show_bug.cgi?id=1509549) [in review](https://bugzilla.mozilla.org/show_bug.cgi?id=1504101) to enable just that, and you can try out builds that include those patches: From [this try push](https://treeherder.mozilla.org/#/jobs?repo=try&revision=6b2b61d055279cb5dff4361fc428460891d6ddd2), download the Firefox Desktop build for your host platform, and connect it to your custom Android build through WebIDE. Before profiling, in the *Performance* panel in WebIDE, open the *Local build* section and add your Android build's objdir to the list. Then profile as usual, and you should be getting full symbol information.
+At the moment, Firefox Nightly Desktop doesn't support symbolicating profiles that were obtained from local Android Gecko builds. However, [patches are](https://bugzilla.mozilla.org/show_bug.cgi?id=1509549) [in review](https://bugzilla.mozilla.org/show_bug.cgi?id=1504101) to enable just that, and you can try out builds that include those patches: From [this try push](https://treeherder.mozilla.org/#/jobs?repo=try&revision=6b2b61d055279cb5dff4361fc428460891d6ddd2), download the Firefox Desktop build for your platform, and connect it to your custom Android build through WebIDE. Before profiling, in the *Performance* panel in WebIDE, open the *Local build* section and add your Android build's objdir to the list. Then profile as usual, and you should be getting full symbol information.
 
 ## Tips
 
 * Avoid opening any other panels besides the Performance panel, to reduce the overhead of activated panels.
 * After connecting to the device, do not select *Main Process* or any of the open tabs from the list of targets on the left.
 * If WebIDE automatically reconnects to a debugging target from a previous session, disconnect, uncheck the two checkboxes in the WebIDE preferences (*Project* > *Preferences*) and reconnect to the device.
-* If the recording doesn't start after clicking the start button, or if the button is disabled or in an otherwise confused state, it might be necessary to close and restart WebIDE to reset some internal state.
+* If the recording doesn't start after clicking the start button, or if the button is inactive or in an otherwise confused state, it might be necessary to close and restart WebIDE to reset some internal state.
```

Comment:
This change is fine. I want to keep the test for `disabled` because we write text about accessibility and might use this word by mistake.